frontends/web/src/components/forms/radio.module.css

```diff
@@ -50,7 +50,7 @@
 /* checked */
 
```

Comment:
there is more in this file, search for `background: rgb(255, 255, 255);`

Reply:
updated to `--background-secondary`